.changeset/new-masks-divide.md

```diff
@@ -0,0 +1,5 @@
+---
+'@shopify/ui-extensions-server-kit': minor
```

Comment:
can we make this a patch?

Reply:
Technically it's a new feature for this library. I can update it though - it's internal use only. Is that needed to not mess with the release?